### PR TITLE
fix: restore mechanism for themes

### PR DESCRIPTION
**Pull request by ssemenkoff**
Added theme restore mechanism for ColorThemePlugin.
Updated vue-epic-bus version to prevent issues with transpileDependencies. 

Connected with this [PR](https://github.com/epicmaxco/vuestic-admin/pull/665)

### CODE REVIEW

File: src/services/ColorThemePlugin.js [hunk 1]
```diff
@@ -19,8 +19,16 @@ export const ColorThemePlugin = {
       Object.assign(defaultOptions.themes, options.themes)
     }
 
+    let themeCache = Object.assign({}, defaultOptions.themes)
+
     Vue.prototype.$themes = defaultOptions.themes
 
+    Vue.prototype.$themes.restoreThemes = () => {
```
Comment (asvae) [inline src/services/ColorThemePlugin.js]: That shouldn't be a public api. Please just implement in component.

Comment (ssemenkoff) [reply src/services/ColorThemePlugin.js]: Fixed in PR in vuestic-admin repository referenced above.

Comment (asvae) [reply src/services/ColorThemePlugin.js]: @ssemenkoff I'm not following? Do I close this PR then?

"vue-epic-bus": "^0.1.3",
This change seems useful.

Comment (ssemenkoff) [reply src/services/ColorThemePlugin.js]: @asvae You can merge it now.

**Pull request merged**